pages/client/application/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Client - View Application</title>
+    <link rel="stylesheet" href="/dist/css/index.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Alfa+Slab+One&family=Montserrat&family=Montserrat+Subrayada&family=Pathway+Gothic+One&family=Roboto+Mono&family=Titillium+Web:wght@900&display=swap" rel="stylesheet" />
+  </head>
+  <body id="homePage">
+    <header></header>
+    <main class="m-5">
+      <!-- Card wrapper-->
+      <div class="h-100 align-items-center d-flex mb-5">
+        <!-- Card -->
+        <div class="card m-auto clientAplication">
+          <img class="card-img-top" id="applicationImage" src="https://via.placeholder.com/450" alt="Application image" />
+          <div class="card-header pt-4">
+            <h3 class="card-title ms-3" id="clientApplicationTitle">Listing Title</h3>
+            <p class="card-text ms-3">
+              <span id="contactPerson"> Contact person </span> / <span id="companyName"> Company name </span> /
+              <span id="loaction"> Location </span>
+            </p>
+          </div>
+          <!-- Card body -->
+          <div class="card-body m-3">
+            <h2 class="d-inline" id="aplicantsName">Applicants name</h2>
+            <p class="d-inline ps-5" id="time">Time/Date</p>
+            <p class="card-text py-3">
+              <span id="clientApplicationEmail"> Email </span>
+              <br />
+              <span id="clientApplicationAddress"> Address </span>
+              <br />
+              <span id="clientApplicationPhone"> Phone number </span>
+            </p>
+
+            <h5 class="font-bold">Lorem</h5>
+
+            <div class="card-text">
+              <p class="mb-2" id="applicationText">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod, nisi id tempus porttitor, risus erat euismod risus, at dapibus neque turpis vel est. Sed accumsan, nibh id euismod bibendum, elit est placerat nibh, euismod rhoncus nisl nisl in est.</p>
+              <br />
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod, nisi id tempus porttitor, risus erat euismod risus, at dapibus neque turpis vel est.</p>
+            </div>
+
+            <h5 class="font-bold my-4" id="apllicantsName">Applicants/Student Name</h5>
+
+            <div class="mb-4">
+              <div>
+                <img src="https://via.placeholder.com/20" alt="Placeholder Icon" />
```

Comment:
Links need a class to remove the green color on the links.
Try adding `class="nav-link text-black"`